<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseSortDialog</class>
 <widget class="QDialog" name="ChooseSortDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VSI</string>
  </property>
  <widget class="QLabel" name="labelChooseSort">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>50</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите сортировку</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>79</y>
     <width>401</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButtonHeapSort">
      <property name="text">
       <string>HeapSort</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonShellSort">
      <property name="text">
       <string>ShellSort</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonQuickSort">
      <property name="text">
       <string>QuickSort</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLoadData">
      <property name="text">
       <string>Выбрать данные</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnGoSort">
      <property name="text">
       <string>Сортировать</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
